src/components/navigation/nav-tab-panel.tsx

```diff
@@ -55,9 +56,20 @@ export class NavTabPanel extends BaseComponent<IProps> {
                 { tabs?.map((tabSpec, index) => {
                     const tabClass = `top-tab tab-${tabSpec.tab}
                                       ${selectedTabIndex === index ? "selected" : ""}`;
+                    const handleSortWorkClick = () => {
+                      closeSubTabDocument("sort-work", "sort-work");
```

Comment:
this is probably another way to do this which is closer to how the other panels work, but I haven't looked.